Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,146 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+## Examples
+
+### Using a `CephClient` to create Ceph credentials
```

Comment:
If we merge this with the above example, no need for this header either

Reply:
- [ ] remove header about using `CephClient` to create creds